Userland/Libraries/LibSQL/Value.h

```diff
@@ -71,6 +72,7 @@ class Value {
     Function<void(ByteBuffer&, size_t &offset)> m_deserialize;
     Function<size_t()> m_size;
     Function<const char *()> m_type_name;
+    Function<bool(Value const&)> m_can_cast;
```

Comment:
Yes. I really wanted to be able to pass Values by reference as stack objects and be able to store them in containers as objects. If I would have used virtual methods this would have been impossible.